index.html

```diff
@@ -93,7 +93,8 @@ <h2>LET'S CONNECT</h2>
       <article class="card" id="work-article-1">
```

Comment:
- [ ] As it is stated in the project requirements, kindly note that the section of the entire project is supposed to be generated dynamically and not have the projects section hardcoded. This you can achieve by having the details of the projects in an array and having the method to generate the projects be called immediately the page is loaded and populate them.

![image](https://user-images.githubusercontent.com/86318284/196886789-2f95dca1-9fbe-4ceb-90ab-4208620f5b25.png)

Reply:
Dear the code is not in Html it is dynamically added from the Javascript Object. Kindly check the HTML file there is only an empty container there thankyou 🤗✔❤

Reply:
OMG you mean I have to create the whole work -->> card section dynamically as well ok I better start working on it than

Reply:
Hello, @studiosnabeel apologies for the late response. Yes you need to create the whole works cards section dynamically.

Reply:
@Nemwel-Boniface is this a new requirement especially for this week's students to create the whole works cards section dynamically?

Reply:
I sent that screenshot from your project requirements. It was there you might just have missed it when doing the project.